### PR TITLE
Fix misc Home Owners issues

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* /bcgov/entity#13726

*Description of changes:*

- Fix styling and other misc bugs for Home Owners

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/views/newMhrRegistration/HomeOwners.vue [hunk 1]
```diff
@@ -179,23 +178,21 @@ export default defineComponent({
...
 
     const localState = reactive({
       showAddPersonSection: false,
       showAddPersonOrganizationSection: false,
-      disableAddHomeOwnerBtn: computed(() => {
-        return localState.showAddPersonOrganizationSection || localState.showAddPersonSection
-      }),
+      disableAddHomeOwnerBtn: computed(
+        () => localState.showAddPersonOrganizationSection || localState.showAddPersonSection
+      ),
```
Comment (cameron-eyds) [inline ppr-ui/src/views/newMhrRegistration/HomeOwners.vue:189]: Not opposed to short form but would stick with whatever format is already in place here. 
If this is the norm for the component than all good 👍 

Just whatever is most consistent :)

**Review by cameron-eyds — APPROVED** [review]
Looks good!

File: ppr-ui/src/views/newMhrRegistration/HomeOwners.vue (hunk 1 above)
Comment (dimak1) [reply ppr-ui/src/views/newMhrRegistration/HomeOwners.vue:189]: Sure thing, just thought I would clean up some redundant code (i.e. `return` statement)

**Review by ChasBoucher — COMMENTED** [review]
Looks good.

File: ppr-ui/src/views/newMhrRegistration/HomeOwners.vue (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/src/views/newMhrRegistration/HomeOwners.vue:189]: Totally! Looks good 👍

**Comment by dimak1** [comment]
> Looks good.

Are you going to approve? @ChasBoucher

**Pull request merged**